modules/TestUtils.js

```diff
@@ -1,5 +1,7 @@
 import isEqual from 'is-equal'
 import isRegExp from 'is-regex'
+import has from 'has'
+import assert from './assert'
```

Comment:
An interesting side effect of making all assertions inline in the `to*` methods is that we don't actually need to `import assert` in this file. :D